MapboxCoreNavigation/RouteController.swift

```diff
@@ -455,19 +455,6 @@ open class RouteController: NSObject {
 
 extension RouteController {
     @objc func progressDidChange(notification: NSNotification) {
-        if let progress = notification.userInfo?[RouteControllerProgressDidChangeNotificationProgressKey] as? RouteProgress {
-            let currentDestinationWaypoint = progress.currentLeg.destination
-
-            if progress.currentLegProgress.userHasArrivedAtWaypoint, sessionState.arrivalTimestamp == nil, currentDestinationWaypoint != previousArrivalWaypoint  {
-                delegate?.routeController?(self, didArriveAt: currentDestinationWaypoint)
-                previousArrivalWaypoint = currentDestinationWaypoint
-                
-                if let lastRouteWaypoint = progress.route.legs.last?.destination, lastRouteWaypoint == currentDestinationWaypoint {
-                    sessionState.arrivalTimestamp = Date()
-                    sendArriveEvent()
```

Comment:
Should we be concerned that `sendArriveEvent()` and `sendDepartEvent()` are being called from different methods now?

Reply:
Moved in https://github.com/mapbox/mapbox-navigation-ios/pull/1013/commits/8ba1cfb0875e8d04c88b14a78e856afa52b38976